data_service/Dockerfile

```diff
@@ -1,8 +1,11 @@
-FROM rust:1.77-alpine
+FROM rust:1.77
```

Comment:
painful to have bigger container but I think it's the only way to get diesel_cli to work properly